templates/UmbracoPackageRcl/wwwroot/package.manifest

```diff
@@ -1,5 +1,5 @@
 {
+    "id": "UmbracoPackage",
     "name": "UmbracoPackage",
-    "versionAssemblyName": "UmbracoPackage",
```

Comment:
Both `umbracopackage` and `umbracopackage-rcl` templates will set the ID, instead of `versionAssemblyName`, since the ID will be used as fallback... Although this might make it less obvious how the package version is determined (either the explicit `version` or automatically from `versionAssemblyName` or `id`), it results in a less verbose manifest and removes the need to duplicate the same ID/name (since for most packages, the ID will be the same as the assembly name).